1 Basic Python Jenkins Pipeline/tests/paralel_run.py

```diff
@@ -0,0 +1,3 @@
+'''pip install pytest-xdist -> in Pycharm "Tools" -> "Sync Python requirements
+pytest -n 3
+'''
```

Comment:
it lacks new line, why is this .py file if it bears a comment only ? Would be better in readme.MD file.